tests/components/iotty/test_init.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for the iotty integration."""
+
+from unittest.mock import MagicMock
+
+from homeassistant.components.iotty.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_entry_oauth2_flow
+
+from tests.common import MockConfigEntry
+
+
+async def test_load_unload_coordinator_called(
```

Comment:
The name of the test, and the use of the `mock_coordinator` fixture suggests we would check the calls to the coordinator, however that doesn't seem to happen?

Reply:
Hi, I have added a few assertions on that, let me know if that satisfies you